### PR TITLE
MANTA-4727 remove "insecure" buckets-api server instance

### PR DESCRIPTION
**Pull request by jlevon**

### CODE REVIEW

File: docs/internal/design.md [hunk 1]
```diff
@@ -40,12 +40,12 @@ purely in-memory so all muskie *processes* have a cached copy.
...
-There is a second server running in each muskie process to
-facilitate monitoring.  This is accessible at `http_insecure_port + 800`.  For
-example, the first (of 16 in production) muskie process within a zone usually
-runs on port 8081, so the monitoring server would be accesible on port 8881 from
-both the `manta` and `admin` networks.  The monitoring server exposes
-Kang debugging information and Prometheus application metrics.
+There is a second server running in each muskie process to facilitate
+monitoring.  This is accessible at `port + 800`.  For example, the first (of 16
+in production) muskie process within a zone usually runs on port 8081, so the
+monitoring server would be accesible on port 8881 from both the `manta` and
```
Comment (kellymclaughlin) [inline docs/internal/design.md]: Not your mistake, but I notice that *accesible* is missing an *s*.

**Review by bahamas10 — DISMISSED** [review]
Pulled this image from jenkins and tested it on my lab - everything works as expected and i verified it is in fact not listening on multiple ports.

```
# svccfg export buckets-api | grep 'port'
<service_bundle type='manifest' name='export'>
    <exec_method name='start' type='method' exec='node --abort-on-uncaught-exception main.js -f etc/config.json -p %{buckets-api/port} &amp;' timeout_seconds='30'>
        <propval name='port' type='astring' value='8081'/>
        <propval name='port' type='astring' value='8082'/>
        <propval name='port' type='astring' value='8083'/>
        <propval name='port' type='astring' value='8084'/>
```

**Review by kellymclaughlin — COMMENTED** [review]
Just a minor nit, but looks good otherwise.

File: docs/internal/design.md (hunk 1 above)
Comment (jlevon) [reply docs/internal/design.md]: Fixed, and also checked the rest of this file for any others (which I didn't find).

**Review by kellymclaughlin — APPROVED** [review]

**Comment by jlevon** [comment]
@bahamas10 you OK with this small update?

**Review by bahamas10 — APPROVED** [review]
LGTM

**Pull request merged**